src/main/resources/db/changelog/db.changelog.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="1" author="andrei_astaschenko">
+        <createTable tableName="pc_order" schemaName="app">
+            <column name="pc_order_id" type="varchar(255)">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="pc_name" type="varchar(255)"/>
```

Comment:
It means 'pet clinic'. I added question about naming convention in PoC documentation.